Getting below error while running Junit cases

this is error we are getting on SOP in Unit test block . But build went successfully.

-------------

13:25:20.238 [main] DEBUG org.reflections.Reflections - expanded subtype org.w3c.dom.Node -> org.w3c.dom.Attr
13:25:20.239 [main] DEBUG org.reflections.Reflections - expanded subtype java.nio.channels.Channel -> java.nio.channels.ReadableByteChannel
13:25:20.239 [main] DEBUG org.reflections.Reflections - expanded subtype java.io.InputStream -> java.io.FileInputStream
13:25:20.239 [main] WARN  org.reflections.Reflections - could not get type for name javax.servlet.SingleThreadModel from any class loader
org.reflections.ReflectionsException: could not get type for name javax.servlet.SingleThreadModel
	at org.reflections.ReflectionUtils.forName(ReflectionUtils.java:390) [reflections-0.9.11.jar:?]
	at org.reflections.Reflections.expandSuperTypes(Reflections.java:381) [reflections-0.9.11.jar:?]
	at org.reflections.Reflections.<init>(Reflections.java:126) [reflections-0.9.11.jar:?]
	at org.arrah.framework.udf.UDFEvaluator.<clinit>(UDFEvaluator.java:34) [classes/:?]
	at org.arrah.org.arrah.framework.udf.UDFEvaluatorTest.test1(UDFEvaluatorTest.java:21) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47) [junit-4.11.jar:?]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit-4.11.jar:?]
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44) [junit-4.11.jar:?]
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271) [junit-4.11.jar:?]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70) [junit-4.11.jar:?]
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229) [junit-4.11.jar:?]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309) [junit-4.11.jar:?]
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252) [surefire-junit4-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141) [surefire-junit4-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112) [surefire-junit4-2.12.4.jar:2.12.4]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189) [surefire-api-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75) [surefire-booter-2.12.4.jar:2.12.4]
Caused by: java.lang.ClassNotFoundException: javax.servlet.SingleThreadModel
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381) ~[?:1.8.0_111]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_111]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331) ~[?:1.8.0_111]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_111]
	at org.reflections.ReflectionUtils.forName(ReflectionUtils.java:388) ~[reflections-0.9.11.jar:?]
	... 33 more
13:25:20.240 [main] DEBUG org.reflections.Reflections - expanded subtype java.lang.reflect.Type -> java.lang.reflect.WildcardType
13:25:20.245 [main] INFO  org.arrah.framework.udf.UDFEvaluator - Indexing all UDFs
13:25:20.258 [main] INFO  org.arrah.framework.udf.UDFEvaluator - {org.arrah.framework.udf.sample.SUM=public java.lang.Long org.arrah.framework.udf.sample.SUM.eval(java.util.ArrayList)}
13:25:20.259 [main] INFO  org.arrah.framework.udf.UDFEvaluator - {org.arrah.framework.udf.sample.UPPER=public java.util.List org.arrah.framework.udf.sample.UPPER.eval(java.util.ArrayList)}
13:25:20.259 [main] INFO  org.arrah.framework.udf.UDFEvaluator - Indexing all UDFs - [OK]
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.147 sec

Results :

Tests run: 47, Failures: 0, Errors: 0, Skipped: 1
----------------------

--- a/pom.xml
+++ b/pom.xml
@@ -108,7 +108,7 @@
         <dependency>
             <groupId>org.reflections</groupId>
             <artifactId>reflections</artifactId>
-            <version>0.9.11</version>
+            <version>0.9.10</version>
         </dependency>
         <dependency>
             <groupId>com.opencsv</groupId>

